runtime/common/src/lib.rs

```diff
@@ -3,3 +3,33 @@
 pub mod constants;
 pub mod extensions;
 pub mod weights;
+
+/// Macro to set a value (e.g. when using the `parameter_types` macro) to either a production value
+/// or to an environment variable or testing value (in case the `frequency-rococo-local` feature is selected or in instant-sealing mode).
+/// Note that the environment variable is evaluated _at compile time_.
+///
+/// Usage:
+/// ```Rust
+/// parameter_types! {
+/// 	// Note that the env variable version parameter cannot be const.
+/// 	pub LaunchPeriod: BlockNumber = prod_or_local_or_env!(7 * DAYS, 1 * MINUTES, "FRQCY_LAUNCH_PERIOD");
+/// 	pub const VotingPeriod: BlockNumber = prod_or_local_or_env!(7 * DAYS, 1 * MINUTES);
+/// }
+/// ```
+#[macro_export]
+macro_rules! prod_or_local_or_env {
+	($prod:expr, $test:expr) => {
+		if cfg!(feature = "frequency-rococo-local") {
+			$test
+		} else {
+			$prod
+		}
+	};
+	($prod:expr, $test:expr, $env:expr) => {
+		if cfg!(feature = "frequency-rococo-local") {
+			core::option_env!($env).map(|s| s.parse().ok()).flatten().unwrap_or($test)
+		} else {
+			$prod
+		}
+	};
+}
```

Comment:
:1st_place_medal:

Reply:
So it looks like the only way to get an environment variable value is if it's configured for frequency-rococo-local, is that the intention?

Reply:
Correct. We wanted to make sure that there was no way to adjust the values for a situation where there was more than one authority. (aka public rococo and mainnet)